File: src/main/java/sk/tuke/gamestudio/game/bricksbreaking/stovka/game/Score.java
package sk.tuke.gamestudio.game.bricksbreaking.stovka.game;

public class Score {

    private int score;

    public void record(int removedTiles) {
        if (removedTiles > 1) {
            score += (removedTiles * 15);
        }
    }

    public int getScore() {
        return score;
    }

    public void recordRound() {
        score += 25000;
    }

}

File: web/src/game/board/score.ts

export class Score {

    private score: number = 0;

    record(removedTiles: number) {
        if (removedTiles > 1) {
            this.score += (removedTiles * 2);
        }
    }

    getScore() : number {
        return this.score;
    }

    recordRound() {
        this.score += 100;
    }

}
